qiskit_nature/properties/second_quantization/electronic/bases/electronic_basis.py

```diff
@@ -20,7 +20,7 @@ class ElectronicBasis(Enum):
 
     This ``Enum`` simply names the available electronic bases. The ``SO`` basis is the _special_
     basis into which an ``ElectronicEnergy`` must map its integrals before being able to perform the
```

Comment:
```suggestion
    This ``Enum`` simply names the available electronic bases. The ``SO`` basis is the *special*
    basis into which an ``ElectronicEnergy`` must map its integrals before being able to perform the
```
I think that was intended to come out in italic font - while the underscore is interpreted that way for markdown text it does not do it for Sphinx